packages/firestore/test/unit/core/query.test.ts

```diff
@@ -218,6 +218,64 @@ describe('Query', () => {
     expect(queryMatches(query1, document)).to.be.true;
   });
 
+  it('matches NOT_IN filters', () => {
```

Comment:
Same comment re NOT_IN vs not-in. NOT_IN is a proto representation detail while `not-in` is the public API value.